Studies/python/Training.py

```diff
@@ -17,6 +17,24 @@
 from CalculateWeigths import CreateSampleWeigts, CrossCheckWeights
 import ROOT
 
+
+# import tensorflow as tf
+# gpus = tf.config.experimental.list_physical_devices('GPU')
+# tf.config.experimental.set_memory_growth(gpus[0], True)
+#
+# from tensorflow import keras
```

Comment:
This is training file for TF1. We are using TF2. Please rename this file to TrainingTF1.py and add Training.py that corresponds to TF2 training.